src/Features/CSharp/Portable/CodeRefactorings/ConvertLocalFunctionToMethod/CSharpConvertLocalFunctionToMethodCodeRefactoringProvider.cs

```diff
@@ -33,7 +33,7 @@ internal sealed class CSharpConvertLocalFunctionToMethodCodeRefactoringProvider(
     public override async Task ComputeRefactoringsAsync(CodeRefactoringContext context)
     {
         var (document, textSpan, cancellationToken) = context;
-        if (document.Project.Solution.WorkspaceKind == WorkspaceKind.MiscellaneousFiles)
+        if (!await document.Project.HasSuccessfullyLoadedAsync(context.CancellationToken).ConfigureAwait(false))
```

Comment:
i would not make this change.  

Note: i think we can remove the check entirely here.

Reply:
reverting this.